slack_bolt/app/app.py

```diff
@@ -339,6 +346,10 @@ def listener_runner(self) -> ThreadListenerRunner:
         """The thread executor for asynchronously running listeners."""
         return self._listener_runner
 
+    @property
+    def process_before_response(self) -> bool:
```

Comment:
Added the field `process_before_response` as the Djando adapter needs to know the configuration.